src/Entity/Rating.php

```diff
@@ -13,45 +13,40 @@ class Rating
      * @ORM\Id()
      * @ORM\GeneratedValue()
      * @ORM\Column(type="integer")
+     * @var int
      */
     private $id;
 
     /**
      * @ORM\Column(type="text")
+     * @var string
      */
     private $text;
 
     /**
      * @ORM\Column(type="smallint")
+     * @var int
      */
     private $value;
 
-    public function getId(): ?int
+    public function __construct(string $text, int $value)
     {
-        return $this->id;
+        $this->text = $text;
+        $this->value = $value;
```

Comment:
premýšľam že ten smallint má nejaký rozsah, kľudne si do konštruktorov môžeš pridať nejaké validácie, že ti tam nechodia príliš veľké čísla, a ak áno tak vyhodíš vynímku `\InvalidArgumentException`... môžeš si do projektu pridať https://packagist.org/packages/beberlei/assert jak máme v práci

Reply:
jo přidám dík

Reply:
možeš aj v iných PR :) to len ako nápad